### DISCUSSION TITLE
更聪明地学习，而不是苦读——《如何高效学习》 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#220] [Announcements]
https://mazhuang.org/2020/05/17/learn-more-study-less/ 

 我们可能都听过一句话：

**Comment by wenhaofree** [comment]
收藏了

**Comment by GoggleHe** [comment]
膜拜12年就在更博的大神

**Comment by mzlogin** [comment]
> @GoggleHe
> 膜拜12年就在更博的大神

:laugh: 跟大神还差得远呐

**Comment by ke-nan** [comment]
请问大佬，如何把文章页的搜索框的宽度调成和它底下元素的宽度一样？

**Comment by mzlogin** [comment]
> @ke-nan
> 请问大佬，如何把文章页的搜索框的宽度调成和它底下元素的宽度一样？

按目前这种结构应该只能用 JavaScript 在页面加载完后动态调整了，我以前尝试过，效果也不是很好就放弃了。

**Comment by HillCat** [comment]
他这本书我认真做了笔记，最大的感受就是，纸质笔记这块要重视，很多笔记流，隐喻，类比，画图 ，抽象，思考这些东西，其实都是需要荧光笔，水性笔，活页纸，这种纸质笔记搞定的。特别是在抽象概念或者深度思考的时候，把零散的知识点连接起来，电子云笔记是很难达到这些目的。

**Comment by mzlogin** [comment]
> 他这本书我认真做了笔记，最大的感受就是，纸质笔记这块要重视，很多笔记流，隐喻，类比，画图 ，抽象，思考这些东西，其实都是需要荧光笔，水性笔，活页纸，这种纸质笔记搞定的。特别是在抽象概念或者深度思考的时候，把零散的知识点连接起来，电子云笔记是很难达到这些目的。

👍 👍 👍
